package.json

```diff
@@ -24,5 +24,8 @@
   "prettier": {
     "trailingComma": "all",
     "endOfLine": "lf"
+  },
+  "dependencies": {
+    "glicko2": "^1.2.1"
```

Comment:
This dependency should be in [`packages/server/package.json`](https://github.com/govariantsteam/govariants/blob/0abf1b20636ce4d87ed3b3e378d598a527affebb/packages/server/package.json#L12), not the top-level `package.json`.

Reply:
The dependency was automatically added when I ran the command `yarn add glicko2`. Should I manually remove it from the top level `package.json` and add it to the `package.json` in the server folder?

Reply:
> The dependency was automatically added when I ran the command yarn add glicko2.

Yeah unfortunately the "monorepo" is a bit confusing. If you are in the server/ directory when you call `yarn add` it would have added it to the correct package file.

> Should I manually remove it from the top level package.json and add it to the package.json in the server folder?

Yep, that's probably the easiest way at this point.